test/e2e/addons/smb-share/smb_share_test.go

```
@@ -71,9 +71,9 @@ var _ = AfterSuite(func(ctx context.Context) {
 	GinkgoWriter.Println("Namespace <", namespace, "> and secret <", secretName, "> deleted on cluster")
 	GinkgoWriter.Println("Checking if addon is disabled..")
 
-	status := suite.K2sCli().GetStatus(ctx)
+	enabled, _ := suite.AddonsInfo().IsAddonEnabled(addonName)
```

Please do not ignore errors.